assets/src/application/ui/display/icons/svgs/index.ts

```diff
@@ -39,3 +44,9 @@ export { ReactComponent as UpCircleFilled } from './upCircleFilled.svg';
 export { ReactComponent as Warning } from './warning.svg';
 
 export { ReactComponent as WarningTriangle } from './warningTriangle.svg';
+
+// Enhanced icons
+
+export const InfoCircleOutlined = pipe(withGeneratedClassName, withSize)(InfoCircleOutlinedSVG);
+
+export const GlobalOutlined = pipe(withGeneratedClassName, withSize)(GlobalOutlinedSVG);
```

Comment:
sry I don't think I was very clear in my comments before...

when I said: 

> can this not just be combined with the logic inside withEnhance ???

what I meant was that having a separate HOC **just** for applying the icon name as a css class to an icon...
as well as **another** separate HOC for applying size and margin related classes to an icon...
seems kind of redundant...

They are both just applying classes to an icon, therefore it seems more logical to me to combine these together into one HOC that applies icon classes.
To go further I **actually** think that typical modifier classes for a component should be part of the component itself and not an HOC which imho should be reserved for applying unrelated and/or atypical changes to a component.

In both of these cases however I think that the enhancements should apply to ALL of our icons:

- adding the icon name as a css class should be applied to ALL of our icons, because that will undoubtedly be handy and allow any icon to be targeted via css. This is similar to how all of the WP dashicons have their icon name as part of their css classes like `dashicons dashicons-calendar`.

- being able to change the size of an icon also seems like something that all icons should have the capacity to do. AFAIK this can be achieved indirectly by modifying the font size for the container, but imho it owul dbe better to have some standardized sizes that could be controlled more centrally

- removing margins also seems pretty generic and could likely be a nice to have for any icon

So bottom line, let's just have a single HOC for adding css modifiers
(and I'm assuming that the HOC is used because the icon components are dynamically generated and there isn't a single `Icon` component where that logic can be centralized?

Reply:
there are custom templates from `svgr` which would apply that HOC to each of the SVG component: https://react-svgr.com/docs/custom-templates/
***
need to think of an easier way to achieve this

Reply:
tried dynamically to apply those HOC, but got no luck:
```
import { pipe } from 'ramda';

import withGeneratedClassName from './svgs/withGeneratedClassName';
import withSize from './svgs/withEnhance';

import * as icons from './svgs';

const ehancedIcons = Object.entries(icons).reduce((acc, [name, component]) => {
	return (acc[name] = pipe(withGeneratedClassName, withSize)(component));
}, {});

export default ehancedIcons;
```